lib/main/localizations/app_localizations.dart

```diff
@@ -3844,4 +3844,18 @@ class AppLocalizations {
       name: 'selectAllMessagesOfThisPage',
     );
   }
+
+  String get replySentSuccessfully {
+    return Intl.message(
+      'Reply sent successfully!',
+      name: 'replySentSuccessfully',
+    );
+  }
+
+  String get replySentUnsuccessfully {
+    return Intl.message(
+      'Reply sent unsuccessfully!',
+      name: 'replySentUnsuccessfully',
```

Comment:
Please contact @hoangdat  to get suitable text

Reply:
Text replaced